eclair-core/src/main/scala/fr/acinq/eclair/payment/send/PaymentLifecycle.scala

```diff
@@ -266,6 +240,49 @@ class PaymentLifecycle(nodeParams: NodeParams, cfg: SendPaymentConfig, router: A
     goto(WAITING_FOR_ROUTE) using WaitingForRoute(data.sender, data.c, data.failures :+ failure, ignore1)
   }
 
+  private def handleUpdate(nodeId: PublicKey, failure: Update, data: WaitingForComplete): Seq[Seq[ExtraHop]] = {
```

Comment:
```suggestion
  /**
   * Apply the channel_update to our routing table
   * 
   * @return Updated routing hints if applicable
   */
  private def handleUpdate(nodeId: PublicKey, failure: Update, data: WaitingForComplete): Seq[Seq[ExtraHop]] = {
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1441/commits/5fa4ae5f146e3c3827ee474ddf555c9b30785aaa